src/clarity-angular/wizard/wizard-page.ts

```diff
@@ -119,6 +119,25 @@ export class WizardPage implements OnInit {
     @Output("clrWizardPagePreventDefaultCancelChange") stopCancelChange: EventEmitter <boolean> =
         new EventEmitter();
 
+    // overrides next from the page level, so you can use an alternate function for
+    // validation or data-munging with clrWizardPageOnNext
+    private _stopNext = false;
+    public get stopNext(): boolean {
+        return this._stopNext;
+    }
+    @Input("clrWizardPagePreventDefaultNext")
+    public set stopNext(val: boolean) {
+        let valBool = !!val;
```

Comment:
This code is carried over from similar implementations in the datagrid. I presume the `!!val` is a guard against a non-boolean being sent by accident.